examples/com/venafi/vcert/sdk/example/policymanagement/PolicyManagementYamlExample.java

```diff
@@ -0,0 +1,70 @@
+package com.venafi.vcert.sdk.policy;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.venafi.vcert.sdk.Config;
+import com.venafi.vcert.sdk.VCertClient;
+import com.venafi.vcert.sdk.endpoint.Authentication;
+import com.venafi.vcert.sdk.endpoint.ConnectorType;
+import com.venafi.vcert.sdk.policy.domain.PolicySpecification;
+
+import java.io.File;
+
+/**
+ * The following test is to show how to use the policy management feature
+ * in order to create/update and get a policy for both TPP and Cloud (OutagePredict)
+ */
+public class PolicyManagementYamlTest {
```

Comment:
Same issue with class name

Reply:
mmmm, I see that the refactoring name IntelliJ action only refactored the name of the file, not the name of the class.

Reply:
it's done